### PR TITLE
cam #16406 Harden camel.ActorProducerTest

### PR DESCRIPTION
**Pull request by patriknw**
Refs #16406

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,7 @@
 .scala_dependencies
 .scalastyle
 .settings
+.cache*
```
Comment (drewhk) [inline .gitignore]: what creates this?

**Comment by drewhk** [comment]
LGTM

**Comment by ktoso** [comment]
LGTM

File: .gitignore (hunk 1 above)
Comment (patriknw) [inline .gitignore]: Eclipse Scala IDE 4.1

**Pull request merged**